docs/source/releases/v1_14_0a0.rst

```diff
@@ -0,0 +1,31 @@
+.. dropdown:: Distribution Statement
+
+ | # # # This source code is protected under the license referenced at
+ | # # # https://github.com/NRLMMD-GEOIPS.
+
+Version 1.14.0a0 (2024-07-12)
+*****************************
+
+* Bug Fixes
+
+  * Fix failing test data CLI unit tests
```

Comment:
I think these can each still be listed individually - we'll just keep reusing the branch, but we can enumerate the changes separately.

Hmm, @biosafetylvl5 that might wreak havoc on the automated release note generator?!  Not sure, depends how it is interfaced.

Reply:
Meaning - I don't think we should just have a single "Bug fixes for workshop" header with all the changes under that single header.  We should, as normal, create separater PRs with separate sections in the release notes for each change we make.  We'll just reuse the Issue to create more branches.  It is how I would liek it now, so if you agree go ahead and resolve.